### ISSUE TITLE
[TSDoc> Search]: Screen reader is not announcing descriptive information for the 'Search by Typesense' link under search dialog.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/rushstack-websites#152]
**Test Environment:**
OS Version: 22H2 (OS Build 25370.1)
Browser: Edge Dev (Version 115.0.1866.1 (Official build) dev (64-bit))
URL: https://tsdoc.org/
Screen reader: Narrator

**Repro Steps:**

1. Open the above URL in edge dev browser and login with v-.
2. Turn on the narrator through Ctrl+ windows+ Enter key.
3. Navigate to the 'Search' button and activate it.
4. Navigate to the 'Search by: Typesense' link through tab key under search dialog.
5. Observe the issue.

**Actual Result:**
Screen reader is not announcing descriptive information for the 'Search by Typesense' link under search dialog.
Screen reader is only announcing as **'Content information footer link search by'.**

**Expected Result:**
Screen reader should announce the information of action performed on activating the 'Toggle word wrap' button under 'Intro'.
Screen reader should announce as 'Word text wrapped' or on renavigation screen reader announce the information as selected such as 'Toggle word wrap button selected'.

Notes:
Same issue repro on 'Toggle word Wrap' button present under '@returns'.

User Impact:
Screen reader user will get impacted as user will not get the information of action performed due to which user will get confused that whether the action is performed or not.

WCAG Reference Link:
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

**Comment by vaishali1397** [comment]
Closing this issue and will track with bug id: https://github.com/microsoft/rushstack-websites/issues/172
